tests/db_functions/text/test_sha384.py

```diff
@@ -0,0 +1,44 @@
+from django.db import connection
+from django.db.models import CharField
+from django.db.models.functions import SHA384
+from django.test import TestCase
+from django.test.utils import register_lookup
+
+from ..models import Author
+
+
+class SHA384Tests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Author.objects.bulk_create([
+            Author(alias='John Smith'),
+            Author(alias='Jordan Élena'),
+            Author(alias='皇帝'),
+            Author(alias=''),
+            Author(alias=None),
+        ])
+
+    def test_basic(self):
+        authors = Author.objects.annotate(
+            sha384_alias=SHA384('alias'),
+        ).values_list('sha384_alias', flat=True).order_by('pk')
+        self.assertSequenceEqual(
+            authors,
+            [
+                '9df976bfbcf96c66fbe5cba866cd4deaa8248806f15b69c4010a404112906e4ca7b57e53b9967b80d77d4f5c2982cbc8',
```

Comment:
Same here - these fit in 119.

(In addition, the line with `if connection.features.interprets_empty_strings_as_nulls else` in is 83 chars for every function, so that already breaks 79 chars.)

Reply:
Agreed, lets leave it.